CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+* Introduce a new version `1` format that better handle `Time` and `DateTime` objects. It can be enabled by setting `Paquito.format_version = 1`.
```

Comment:
```suggestion
* Introduce a new version `1` format that better handles `Time` and `DateTime` objects. It can be enabled by setting `Paquito.format_version = 1`.
```